class.jetpack-jitm.php

```diff
@@ -98,6 +126,88 @@ function photon_msg() {
 		}
 	}
 
+	/**
+	 * Display message prompting user to enable auto-updates in WordPress.com.
+	 *
+	 * @since 3.8.2
+	 */
+	function manage_pi_msg() {
+		if ( current_user_can( 'jetpack_manage_modules' ) ) {
+			$normalized_site_url = Jetpack::build_raw_urls( get_home_url() );
+			$manage_active = Jetpack::is_module_active( 'manage' );
+			$manage_pi_dismissed = isset( self::$jetpack_hide_jitm['manage-pi'] );
+
+			if ( ! $manage_active || ! $manage_pi_dismissed ) :
+			?>
+			<div class="jp-jitm">
+				<a href="#"  data-module="manage-pi" class="dismiss"><span class="genericon genericon-close"></span></a>
+				<div class="jp-emblem">
+					<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="0" y="0" viewBox="0 0 172.9 172.9" enable-background="new 0 0 172.9 172.9" xml:space="preserve">
+						<path d="M86.4 0C38.7 0 0 38.7 0 86.4c0 47.7 38.7 86.4 86.4 86.4s86.4-38.7 86.4-86.4C172.9 38.7 134.2 0 86.4 0zM83.1 106.6l-27.1-6.9C49 98 45.7 90.1 49.3 84l33.8-58.5V106.6zM124.9 88.9l-33.8 58.5V66.3l27.1 6.9C125.1 74.9 128.4 82.8 124.9 88.9z"/>
+					</svg>
+				</div>
+				<?php if ( ! $manage_active ) : ?>
+					<p class="msg">
+						<?php _e( 'Save time with automated plugin updates.', 'jetpack' ); ?>
+					</p>
+					<p>
+						<img class="j-spinner hide" src="<?php echo esc_url( includes_url( 'images/spinner-2x.gif' ) ); ?>" alt="<?php echo esc_attr__( 'Loading...', 'jetpack' ); ?>" /><a href="#" data-module="manage" data-module-success="<?php esc_attr_e( 'Success!', 'jetpack' ); ?>" class="activate button"><?php esc_html_e( 'Activate remote management', 'jetpack'	); ?></a>
+					</p>
+				<?php elseif ( $manage_active ) : ?>
+					<p>
+						<?php esc_html_e( 'Save time with auto updates on WordPress.com', 'jetpack' ); ?>
+					</p>
+				<?php endif; // manage inactive ?>
+				<?php if ( ! $manage_pi_dismissed ) : ?>
+					<p class="show-after-enable <?php echo $manage_active ? '' : 'hide' ; ?>">
+						<a href="<?php echo esc_url( 'https://wordpress.com/plugins/' . $normalized_site_url ); ?>" target="_blank" title="<?php esc_attr_e( 'Go to WordPress.com to enable auto-updates for plugins', 'jetpack' ); ?>" data-module="manage-pi" class="button button-jetpack launch show-after-enable"><?php esc_html_e( 'Enable auto-updates on WordPress.com', 'jetpack' ); ?></a>
+					</p>
+				<?php endif; // manage-pi inactive ?>
+			</div>
+			<?php
+			//jitm is being viewed, track it
+			$jetpack = Jetpack::init();
+			$jetpack->stat( 'jitm', 'manage-pi-viewed-' . JETPACK__VERSION );
+			$jetpack->do_stats( 'server_side' );
+			endif; // manage or manage-pi inactive
+		}
+	}
+
+	/**
+	 * Display message in editor prompting user to compose entry in WordPress.com.
+	 *
+	 * @since 3.8.2
+	 */
+	function editor_msg() {
+		$post_type = get_post_type();
```

Comment:
get_post_type is supposed to be used <a href="https://codex.wordpress.org/Function_Reference/get_post_type">within the loop</a>.  I'd suggest getting the post type with the global `$typenow` from within the admin.